pkg/argo/fetch.go

```diff
@@ -144,6 +158,21 @@ func Fetch(req *pb.FetchRequest) (*pb.FetchResponse, error) {
 	return newFetchResponse(newFetchRequest(req.Job.Id, stepGroupName, newStepPhase), eof, logs), nil
 }
 
+func isHTMLCode(log string) bool {
+	return strings.HasPrefix(log, "<div")
+}
+
+func snipPodLogs(podLogs []string) ([]string, error) {
+	snipLogs := []string{}
+	for _, log := range podLogs {
+		if isHTMLCode(log) {
+			snipLogs = append(snipLogs, log)
+		}
+		// TODO(yancey1989): snip query result from logs
+	}
+	return snipLogs, nil
```

Comment:
Whether it will appear in the pod logs like:
```html
<div>
   2nd line
</div>
```

Reply:
May not, here follows:
https://github.com/sql-machine-learning/sqlflow/blob/eed73c55f9531279263dca36a5139275b6abc5ce/pkg/sql/submitter.go#L214

but I think more complex HTML code would be used in the future.